src/features/game/lib/landDataStatic.ts

```
@@ -33,9 +33,14 @@ export const STATIC_OFFLINE_FARM: GameState = {
     "Aging Shed": new Decimal(1),
     "Basic Land": new Decimal(30),
   },
+  bumpkin: { 
```

ESLint

[error] Delete `·`

(prettier/prettier)